README.md

```diff
@@ -22,3 +22,106 @@ add script "dev": "next" in the package.json
 yarn install
 yarn run dev
 ```
+
+## Integration tests
+
+This code uses [Watai](https://github.com/MattiSG/Watai/wiki) framework for integration tests.
+
+### Install integration tests environment
+
+To run the tests with Watai, we need Java JRE and a Selenium standalone server.
+
+#### Install Java JRE
+
+Check if a JRE is already installed on your environment with:
+```
+java -version
+# Expected answer example: java version "1.8.0_40"
+```
+
+If you get an error, download Java last `JRE` version on [Oracle website](http://www.oracle.com/technetwork/java/javase/downloads/index.html). 
```

Comment:
On the website, the package to download is called JDK

Reply:
Updated with a direct link to JRE 9 
> No specific risk identified as it works with a Java Development Kit that contains a Java Runtime Environment.